src/platform/packages/shared/react/kibana_mount/mount_point_portal.tsx

```diff
@@ -52,6 +54,9 @@ export const MountPointPortal: FC<PropsWithChildren<MountPointPortalProps>> = ({
         setShouldRender(false);
         el.current = undefined;
       });
+      if (unsetMountPoint) {
+        unsetMountPoint();
```

Comment:
Small improvement to allow to use this component nicer with pattern like `chrome.setAppendBreadcrumbExtension`